langchain_opendataloader_pdf/document_loaders.py

```
@@ -245,48 +255,60 @@ def lazy_load(self) -> Iterator[Document]:
             )
 
         try:
-            output_dir = tempfile.mkdtemp(dir=tempfile.gettempdir())
+            output_dir = tempfile.mkdtemp()
+        except OSError as e:
+            logger.error(f"Failed to create temp directory: {e}")
```

Ruff (0.15.9)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)